test/functionality/dataflow/graph/equal-tests.ts

```diff
@@ -29,23 +29,22 @@ describe('Equal', () => {
 		}
 
 		eq('Empty graphs', emptyGraph(), emptyGraph())
-		eq('Same vertex', emptyGraph().use('0', 'x'), emptyGraph().use('0', 'x'))
+		eq('Same vertex', emptyGraph().use('0'), emptyGraph().use('0'))
 	})
 	describe('Negative', () => {
 		function neq(name: string, a: DataflowGraph, b: DataflowGraph) {
 			raw(name, a, b, 'should differ', x => assert.isFalse(x))
 		}
 		describe('More elements', () => {
-			neq('Additional root vertex', emptyGraph(), emptyGraph().use('0', 'x'))
-			neq('Additional non-root vertex', emptyGraph(), emptyGraph().use('0', 'x', {}, false))
+			neq('Additional root vertex', emptyGraph(), emptyGraph().use('0'))
+			neq('Additional non-root vertex', emptyGraph(), emptyGraph().use('0', {}, false))
 			neq('Additional edge', emptyGraph(), emptyGraph().reads('0', '1'))
 		})
 		describe('Different elements', () => {
 			describe('Different vertices', () => {
-				const rhs = emptyGraph().use('0', 'x')
-				neq('Id', emptyGraph().use('1', 'x'), rhs)
-				neq('Name', emptyGraph().use('0', 'y'), rhs)
-				neq('Control Dependency', emptyGraph().use('0', 'x', { controlDependencies: ['1'] }), rhs)
+				const rhs = emptyGraph().use('0')
```

Comment:
i am still a little bit torn. now it is borderline impossible to link the implementation to the code without at least generating a mermaid-graph representation. 
This should be definitely addressed within #626. But in general, seeing it this way, i am more of a fan of keeping the names within the builder (we can still forcefully attach a `name` during the construction, and then have the diff checker check if a `name` is a attached and if so, if it is equal to the recovered lexeme - just so we keep the additional sanity check of linking ids to lexemes as well.